src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -65,7 +65,22 @@ export class List extends Setup {
             );
     }
 
-    public async iOSDeviceList(): Promise<any> {
```

Comment:
The diff here is misleading. All I did was to make `iOSDeviceList` and `androidDeviceList` to be `private` (since they are internal to this class) and moved them below the `protected` methods.